Mlem/API/APIClient.swift

```diff
@@ -50,6 +51,11 @@ class APIClient {
             // the API, however we may way to consider adding the error model type as an
             // associated value in the same was as the response to allow requests to define
             // their own error models when necessary, or drop back to this as the default...
+            
+            if apiError.error == "not_logged_in" {
```

Comment:
given we get a `400` when it should likely be a `401` I'm just looking at the error string itself, which is disgusting but it's the best the API gives us as it stands... we'll need to keep an eye on this in future Lemmy releases in case they decide to change this messaging.